modules/currencies/currencies.model.js

```diff
@@ -0,0 +1,12 @@
+const currencyType = `    
```

Comment:
Please, rename file to the `currency.graphql.js`

Reply:
done